x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -802,6 +810,110 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     return this.esClient.search<SafeEndpointEvent>(query);
   }
 
+  public async fetchValueListMetaData(interval: number) {
+    if (this.esClient === undefined || this.esClient === null) {
+      throw Error('elasticsearch client is unavailable: cannot retrieve diagnostic alerts');
+    }
+
+    const listQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: '.lists-*',
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        aggs: {
+          total_value_list_count: {
+            cardinality: {
+              field: 'name',
+            },
+          },
+          type_breakdown: {
+            terms: {
+              field: 'type',
+            },
+          },
+        },
+      },
+    };
+    const itemQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: '.items-*',
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        aggs: {
+          value_list_item_count: {
+            terms: {
+              field: 'list_id',
+            },
+          },
+        },
+      },
+    };
+    const exceptionListQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: `${this.kibanaIndex}*`,
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        query: {
+          bool: {
+            must: [
+              { match: { 'exception-list.entries.type': 'list' } },
+              { match: { 'exception-list.entries.operator': 'included' } },
```

Comment:
```suggestion
```
Lists can also use the `excluded` operator, so I think we can leave out the `operator` clause in this query.